aphrodite/modeling/layers/linear.py

```diff
@@ -26,6 +26,69 @@ def adjust_marlin_shard(param, shard_size, shard_offset):
     return shard_size * marlin_tile_size, shard_offset * marlin_tile_size
 
 
+# Split the exl2 weight at group boundary
```

Comment:
Why is this moved to this file?

Reply:
It would be a circular import otherwise.